waku/waku_filter_v2/protocol.nim

```diff
@@ -228,16 +232,20 @@ proc handleMessage*(
     if not await wf.pushToPeers(subscribedPeers, messagePush).withTimeout(
       MessagePushTimeout
     ):
-      debug "timed out pushing message to peers",
+      info "timed out pushing message to peers",
```

Comment:
Isnt it an error? Or do we just want to notice this?

Reply:
> Isnt it an error? Or do we just want to notice this?

ah yes good point! I will change it to `error`